### PR TITLE
ci: test and benchmark radosgw

### PR DESCRIPTION
**Pull request by m-ildefons**
- Run more extensive tests as part of the PR validation worklow
- Run benchmarks after pushes (merges) to branch s3gw

Signed-off-by: Moritz Röhrich <moritz.rohrich@suse.com>

### CODE REVIEW

**Review by jhmarina — DISMISSED** [review]

**Review by jecluis — APPROVED** [review]
lgtm, but requires https://github.com/aquarist-labs/s3gw-tools/pull/164 to be merged first

**Review by jecluis — DISMISSED** [review]
good to merge once its sibling PR on s3gw-tools is merged

File: .github/workflows/test-radosgw.yml
Comment (jecluis) [inline .github/workflows/test-radosgw.yml:720]: do we really want to have these here?

**Review by jecluis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
